File: 02.PYTHON/pg-166-169/pg_166_13.py
print("")
print("**************************************************************************************************")
print("READ TWO INTEGER NUMBERS WITH TWO DIGITS AND DETERMINE IF THE SUM OF BOTH ORIGINATE AN EVEN NUMBER")
print("**************************************************************************************************")
print("")

num1 = int(input("Write the first number: "))
num2 = int(input("Write the second number: "))

if(num1 < 0):
    num1 = num1 * (-1)
if(num2 < 0):
    num2 = num2 * (-1)

if(num1 >= 10 and num1 <= 99):
    if(num2 >= 10 and num2 <= 99):

        result = num1 + num2

        if(result == result // 2 * 2):
            print("The result {} is an even number.".format(result))    
        else:
            print("The result {} is an odd number.".format(result))

    else:
        print("Some number does not have two digits. Try again please.")

else:
    print("Some number doesn't have two digits. Try again please.")

File: 06.PHP/pg-166-169/pg_166_13.php
<?php

echo("");
echo("PHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPH<br>");
echo("READ TWO TWO-DIGIT INTEGERS AND DETERMINE IF THE SUM OF BOTH INTEGERS PRODUCE AN EVEN NUMBER<br>");
echo("PHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPH<br><br>");
echo("");

echo("Write the first integer: ");
$num1 = 73;
echo($num1."<br>");

echo("Write the first integer: ");
$num2 = 12;
echo($num2."<br><br>");

if($num1 < 0)
    {
        $num1 *= -1;
    }
if($num2 < 0)
    {
        $num2 *= -1;
    }

if(($num1 >= 10 && $num1 <= 99) && ($num2 >= 10 && $num2 <= 99))
    {
        $sum = $num1 + $num2;

        if($sum % 2 == 0)
            {
                echo("THE SUM OF BOTH INTEGERS (".$sum.") PRODUCES AN EVEN NUMBER");
            }
            else
                {
                    echo("THE SUM OF BOTH INTEGERS (".$sum.") DOESN'T PRODUCE AN EVEN NUMBER");
                }
    }
    else
        {
            echo("Some of the written numbers doesn't have two digits. Pleas try again!");
        }
?>